kr-core/src/main/clojure/edu/ucdenver/ccp/kr/sparql.clj

```diff
@@ -404,7 +439,14 @@
      (apply str (interleave (map sparql-query-body triple-pattern)
                             ;;(repeat ". \n")))
                             (repeat " \n")))
-   (sparql-operator? (first triple-pattern)) (filter-body triple-pattern)
+   ;; used as a catch-all; this routes anything unusual to a FILTER.
```

Comment:
I found this comment confusing, since (I think, correct me if I'm wrong) the _this_ that it refers to is the line that is commented out.  Since that functionality has been superseded, I'd suggest just removing both code and comment.  If you put a good explanation in the commit log, that will make it clear in future why this changes was necessary and how it differs from the old behaviour.